.gitmodules

```diff
@@ -2,16 +2,16 @@
 	path = _1327/static/bootstrap
 	url = https://github.com/twbs/bootstrap.git
 [submodule "puppetlabs-concat"]
-	path = deployment/modules/concat
+	path = deployment/testing_environment/modules/concat
```

Comment:
just to make sure, s/testing_environment/development/?

...can't believe i just... ah nevermind....